src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Testing/TestRunner.TestRunHandler.cs

```diff
@@ -138,31 +141,83 @@ private static string CreateTestCaseReportMessage(TestRunChangedEventArgs? testR
             {
                 var messageBuilder = new StringBuilder();
                 messageBuilder.Append($"[{result.Outcome}] {result.TestCase.DisplayName}");
-                if (result.ErrorMessage != null || result.ErrorStackTrace != null)
-                {
-                    messageBuilder.AppendLine();
-                }
 
                 if (!string.IsNullOrWhiteSpace(result.ErrorMessage))
                 {
+                    messageBuilder.AppendLine();
                     messageBuilder.AppendLine(IndentString($"{LanguageServerResources.Message}:", 4));
                     messageBuilder.AppendLine(IndentString(result.ErrorMessage, 8));
                 }
 
                 if (!string.IsNullOrWhiteSpace(result.ErrorStackTrace))
                 {
+                    messageBuilder.AppendLine();
                     messageBuilder.AppendLine(value: IndentString($"{LanguageServerResources.Stack_Trace}:", 4));
                     messageBuilder.AppendLine(IndentString(result.ErrorStackTrace, 8));
                 }
 
+                var standardOutputMessages = GetTestMessages(result.Messages, TestResultMessage.StandardOutCategory);
```

Comment:
do you have a screenshot of what it looks like?

Reply:
I updated the description with some output. Let me see if I can get output and error messages.

Reply:
Ah, ok. So just one new line at the top and we squish everything together.
```
Starting test run
[Failed] Microsoft.CodeAnalysis.Tools.Tests.Analyzers.ThirdPartyAnalyzerFormatterTests.TestLoadingAllAnalyzers_LoadsDependenciesFromAllSearchPaths
    Message:
        System.Exception : This is an error message

    Stack Trace:
           at Microsoft.CodeAnalysis.Tools.Tests.Analyzers.ThirdPartyAnalyzerFormatterTests.TestLoadingAllAnalyzers_LoadsDependenciesFromAllSearchPaths() in c:\Users\jorobich\source\format\tests\Analyzers\ThirdPartyAnalyzerFormatterTests.cs:line 218
        --- End of stack trace from previous location ---

    Standard Output Messages:
          Determining projects to restore...
          All projects are up-to-date for restore.

==== Summary ====
Failed!  - Failed:    1, Passed:    0, Skipped:    0, Total:    1, Duration: 9s
```

Reply:
lgtm!

Reply:
Updated